packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -414,5 +414,27 @@ void main() {
 
     expect(value, isFalse);
   });
+  testWidgets('Switch is opaque when disabled', (WidgetTester tester) async {
+    final Key switchKey = UniqueKey();
```

Comment:
Please add another test to check that the switch is not translucent when enabled.

Reply:
Would you prefer if I did something like this?:
`
          child: CupertinoSwitch(
            key: UniqueKey(),
            value: false,
            dragStartBehavior: DragStartBehavior.down,
            onChanged: null,
          ),
`
If I remove the key completely it creates a warning